packages/interactor/src/index.ts

```diff
@@ -5,7 +5,6 @@ export { Page } from './page';
 export { App } from './app';
 export { perform } from './perform';
 export { fillIn } from './fill-in';
-export { focused, focus, blur } from './focused';
```

Comment:
Can we deprecate these first instead of removing them? I say this because we're still in the process of upgrading the FOLIO interactors and the backwards compatibility is actually really nice since we're able to clearly say what is what. If we hold off breaking, then we can continue to enhance the test suite with features like #847 and consume new versions, while still not having everything converted to the new api.